ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -58,6 +60,32 @@ - (void)loadRequest:(NSURLRequest *)request
   if (request.URL && _sendCookies) {
     NSDictionary *cookies = [NSHTTPCookie requestHeaderFieldsWithCookies:[[NSHTTPCookieStorage sharedHTTPCookieStorage] cookiesForURL:request.URL]];
     if ([cookies objectForKey:@"Cookie"]) {
+
+      NSMutableString* script = [[NSMutableString alloc] init];
+      
+      // Get the currently set cookie names in javascriptland
+      [script appendString:@"var cookieNames = document.cookie.split('; ').map(function(cookie) { return cookie.split('=')[0] } );\n"];
```

Comment:
Just a small nit: comments are useful in reading code [especially in quick read throughs in PRs] but are informal. I'd encourage removing them because it forces you to make your code more readable (though there aren't any concerns here). Again, mostly a personal opinion.